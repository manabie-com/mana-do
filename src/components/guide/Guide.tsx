import React from "react";
import "./styles.scss";

const Guid = () => {
  return (
    <div className="guide">
      <i className="guide__text">
        * Double-click to the label to edit the corresponding task!
      </i>
    </div>
  );
};

export default Guid;
