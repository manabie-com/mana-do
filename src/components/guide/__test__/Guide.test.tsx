import { render, screen } from "@testing-library/react";
import React from "react";
import Guide from "../Guide";

const renderComponent = () => {
  return render(<Guide />);
};

describe("<Guide /> rendering", () => {
  it("should match snapshot", () => {
    const wrapper = renderComponent();
    expect(wrapper.container).toMatchSnapshot();
  });

  it("should render UI correctly", () => {
    renderComponent();

    const titleElement = screen.getByText(
      "* Double-click to the label to edit the corresponding task!"
    );

    expect(titleElement).toBeInTheDocument();
    expect(titleElement).toHaveClass("guide__text");
  });
});
