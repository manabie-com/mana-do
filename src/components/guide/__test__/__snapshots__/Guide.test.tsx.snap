// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Guide /> rendering should match snapshot 1`] = `
<div>
  <div
    class="guide"
  >
    <i
      class="guide__text"
    >
      * Double-click to the label to edit the corresponding task!
    </i>
  </div>
</div>
`;
